api/circulation.py

```diff
@@ -268,7 +282,7 @@ def parse_fines(fines):
             # We successfuly secured a loan.  Now create it in our
             # database.
             __transaction = self._db.begin_nested()
-            loan, is_new = licensepool.loan_to(
+            loan, new_loan_record = licensepool.loan_to(
```

Comment:
Renaming to avoid confusion between whether the loan itself is new, vs. whether the loan's data model object already exists in the database.